srcpkgs/io.elementary.files/template

```diff
@@ -14,7 +14,7 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/elementary/files"
 distfiles="https://github.com/elementary/files/archive/${version}.tar.gz"
-checksum=3fa4a2a303c1f0f5f6043a2ae0dca3e101c4aafff96639736cc689390682de3c
+checksum=987f87df2a74c97cb866b34a20ea395ae1f67d03c9517185494860653fbc065c
 
```

Comment:
Duplciate `libgee08-devel` in -devel depends

Reply:
fixed